tdp/cli/commands/plan/dag.py

```diff
@@ -52,7 +52,21 @@ def _validate_filtertype(ctx, param, value):
     is_flag=True,
     show_default=True,
     default=False,
-    help="Whether start operations should be replaced by restart operations.",
+    help="Replace 'start' operations by 'restart' operations.",
+)
+@click.option(
+    "--reverse",
+    is_flag=True,
+    show_default=True,
+    default=False,
+    help="Reverse the final list of operations.",
+)
+@click.option(
+    "--stop",
+    is_flag=True,
+    show_default=True,
+    default=False,
+    help="Replace 'start' operations by 'stop' operations.",
```

Comment:
Add a suggestion to combine with `--reversed`.
```suggestion
    help="Replace 'start' operations by 'stop' operations. This option should be used with --reversed.",
```

Reply:
Done.